### ISSUE TITLE
semantic release v20 support

### ISSUE THREAD

**Issue by sebbo2002** [amanda-mitchell/semantic-release-npm-multiple#176]
Since about two weeks semantic-release v20 is the current version. Is there already support planned for this new version? If so, is there anything I can help you with? In the `package.json` only `19.x` is explicitly specified as peer a dependency.

**Comment by thesuavehog** [comment]
I, also, would appreciate the bump to `semantic-release@20`

**Comment by jhnns** [comment]
In the meantime, you can use [@semantic-release/exec](https://github.com/semantic-release/exec) to cross-publish your package to a different registry. This is my config for the Github registry:

```json
[
  "@semantic-release/exec",
  {
    "verifyConditionsCmd": "echo //npm.pkg.github.com/:_authToken=${process.env.GITHUB_TOKEN} > temp.npmrc && npm whoami --userconfig temp.npmrc --registry https://npm.pkg.github.com/",
    "publishCmd": "npm publish --userconfig temp.npmrc --tag ${nextRelease.channel} --registry https://npm.pkg.github.com/ --no-git-tag-version",
    "successCmd": "rm temp.npmrc",
    "failCmd": "rm temp.npmrc"
  }
]
```

**Comment by amanda-mitchell** [comment]
@sebbo2002, @thesuavehog, @jhnns: Fixed in 3.10!
